Magento_Catalog/styles/_module.scss

```diff
@@ -1 +1,2 @@
 @import 'modules/fotorama';
+@import 'modules/image-ratio';
```

Comment:
I'm not sure that this is the best place to import these styles... I saw that this class is used in `/Magento_Theme/templates/html/picture.phtml` so it has global usage. Maybe some mixin in Atoms would be better?

Reply:
I moved styles to components -> molecules/image, I'll implement this solution in components in related task: https://dev.snowdog.pro/issues/71300